app/services/edition_publisher.rb

```diff
@@ -41,12 +41,18 @@ def fire_transition!
     delete_unpublishing!
   end
 
+  def previous_editions
+    Edition
+      .unscoped
```

Comment:
Are you sure you want to `unscoped` this?  That would include soft-deleted editions.

Reply:
The `.published` means you get `.where(state: :published)` so it won't included soft-deleted editions.

Reply:
Gotcha.  What scope are we removing in that case?

Reply:
`.where.not(state: :deleted)` - I imagine (or hope) it would get optimised away by the DB, but I  don't like using or relying on default scopes so I decided to include the `.unscoped`.